tests/kafkatest/services/streams.py

```diff
@@ -322,11 +322,20 @@ class StreamsEosTestBaseService(StreamsTestBaseService):
 
     clean_node_enabled = True
 
-    def __init__(self, test_context, kafka, command):
+    def __init__(self, test_context, kafka, processing_guarantee, command):
         super(StreamsEosTestBaseService, self).__init__(test_context,
                                                         kafka,
                                                         "org.apache.kafka.streams.tests.StreamsEosTest",
                                                         command)
+        self.PROCESSING_GUARANTEES = processing_guarantee
+
+    def prop_file(self):
+        properties = {streams_property.STATE_DIR: self.PERSISTENT_ROOT,
+                      streams_property.KAFKA_SERVERS: self.kafka.bootstrap_servers(),
+                      streams_property.PROCESSING_GUARANTEES: self.PROCESSING_GUARANTEES}
```

Comment:
We know set the processing guarantee via the properties file (that allows us to easily parametrize the test)